res/css/structures/_TabbedView.scss

```diff
@@ -27,7 +27,9 @@ limitations under the License.
     right: 0;
     margin-top: 8px;
 }
-
+.mx_AuthFooter_Image_Tabbed{
+height: 120px!important;
```

Comment:
Is !important required here ... it should only be used as a last resort.  NOTE: this change doesn't have to move to aether-host-files as it is adding new code and not making any updates.

Reply:
Done